src/components/Navbar/Navbar.tsx

```diff
@@ -8,15 +10,8 @@ export const Navbar = () => {
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">Home</a>
-
-          <a
-            aria-current="page"
-            className="navbar-item has-background-grey-lighter"
-            href="#/people"
-          >
-            People
-          </a>
+          <PageNavLink to="/" text="Home" />
+          <PageNavLink to="/people" text="People" aria-current="page" />
```

Comment:
clicking this link resets search params
![image](https://user-images.githubusercontent.com/117474961/221190528-d16c0c6c-f421-414e-a39a-1c8a1540994c.png)